linux_os/guide/system/accounts/accounts-session/accounts_tmout/ansible/sle12.yml

```diff
@@ -13,3 +13,4 @@
   file:
     path: /etc/profile.d/autologout.sh
     mode: '0755'
+  when: lookup('fileglob', '/etc/profile.d/autologout.sh', errors='warn')
```

Comment:
You are not using any wildcard here in the lookup plugin, so it would be better to use `file` instead of `fileglob`, or using another `stat` task.

It is an interesting approach here. Is your intention to explicitly report a "warning" when the file is not present? Otherwise I would recommend to use `stat` module to check the file existence and use its result in the conditional instead of calling the `lookup` directly.
Here is one example: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/ansible/shared.yml#L9

Reply:
Did you have opportunity to check this comment?